enterprise/server/backends/userdb/userdb_test.go

```diff
@@ -1131,6 +1131,14 @@ func TestUserOwnedKeys_CreateAndUpdateCapabilities(t *testing.T) {
 		{Name: "Developer_ACWrite_Fail", Role: role.Developer, Capabilities: []akpb.ApiKey_Capability{akpb.ApiKey_CACHE_WRITE_CAPABILITY}, OK: false},
 		{Name: "Admin_Executor_Fail", Role: role.Admin, Capabilities: []akpb.ApiKey_Capability{akpb.ApiKey_REGISTER_EXECUTOR_CAPABILITY}, OK: false},
 		{Name: "Developer_Executor_Fail", Role: role.Developer, Capabilities: []akpb.ApiKey_Capability{akpb.ApiKey_REGISTER_EXECUTOR_CAPABILITY}, OK: false},
+		// Even admins should not be able to attach ORG_ADMIN caps to user-owned
+		// keys (for now, we only support setting cache caps on user-owned keys)
```

Comment:
nit: I think "caps" is a little confusing and i'd prefer just using "capabilities" everywhere for clarity